payments/admin.py

```diff
@@ -193,8 +202,19 @@ class OrderInline(admin.StackedInline):
     exclude = ("_product_content_type", "_lease_content_type")
 
 
-admin.site.register([WinterStorageProduct, OrderLine, OrderLogEntry])
+class OrderLogEntryAdmin(admin.ModelAdmin):
+    list_display = ("id", "order_id", "from_status", "to_status", "created_at")
+    list_filter = ("from_status", "to_status")
+    search_fields = ("order__id",)
```

Comment:
A customer name search field could probably come in handy here.

Reply:
I'll add it to the `OrderAdmin` one 👍  then you can see the log entry as inlines on the `Order` page

Reply:
Done
<img width="2216" alt="image" src="https://user-images.githubusercontent.com/15201480/102893886-eee2df80-446a-11eb-9628-47e72986961f.png">

Reply:
👍